calcom/cal.com
# HeadlessUI to Radix

## What does this PR do?

Fixes #2073 

## Type of change


## How should this be tested?

 - Go to Event type page in your mobile phone and see the mobile-specific dropdown to be Radix and have UI/UX consistency to the rest of menus 👇
<img width="286" alt="Event_Types___Cal_com" src="https://user-images.githubusercontent.com/467258/157895198-76683ecc-f834-4e8b-8dfa-cc636c5c0e6e.png">

- Go to Bookings page in your mobile phone and see the mobile-specific dropdown for a booking to be Radix and have UI/UX consistency to the rest of menus 👇
<img width="288" alt="Bookings___Cal_com" src="https://user-images.githubusercontent.com/467258/157961141-83c318fa-40da-405b-9e31-f6af5efe62ee.png">
